### PR TITLE
Fix potential Null pointer dereference in test-closed.c

### PR DESCRIPTION
**Pull request by Coeur**
Supersedes #1613

### CODE REVIEW

File: test/test-closed.c [hunk 1]
```diff
@@ -81,6 +82,7 @@ main(int argc, char **argv)
 	   supports EV_FEATURE_EARLY_CLOSE
 	*/
 	cfg = event_config_new();
+	assert(cfg);
```
Comment (azat) [inline test/test-closed.c:85]: `assert` does nothing in release build, consider replace it with `perror` + `goto err`

Comment (Coeur) [reply test/test-closed.c:85]: We already use this assert in regress.c, in `test_methods`.

And for test-closed, `event_config_new` is the first `calloc` of a `main` function: it's hardly ever going to fail, unless the whole system is about to panic/reboot.
In my opinion, the pull requests made by icy17 were to silence warnings, not to deal with all possible `malloc`/`calloc` of the code.

Comment (azat) [reply test/test-closed.c:85]: >We already use this assert in regress.c, in test_methods.

This is tests, it is a separate group, and they should `tt_assert` instead.

>And for test-closed, event_config_new is the first calloc of a main function: it's hardly ever going to fail, unless the whole system is about to panic/reboot.

If we are writing some checks, they should be generic, it is not hard to fix.

Comment (Coeur) [reply test/test-closed.c]: Anyway, not a big issue, here it is: e25b829ba816e00ef8fe83a6c7f9c65fe79b8b28

**Review by azat — APPROVED** [review]

**Comment by azat** [comment]
CI failed, fix - 66428fb9

**Pull request merged**